apps/backend/src/schemas/configuration.py

```
@@ -0,0 +1,549 @@
+        ):
+            # High risk processed events should have notifications
+            pass  # Allow but may warn in logs
+
+        # Validate change summary consistency
+        if self.changes_summary and self.change_type:
+            if self.change_type == "deleted" and self.changes_summary:
+                # Deleted files shouldn't have detailed changes
+                if (
+                    len(self.changes_summary) > 1
+                    or "deleted" not in str(self.changes_summary).lower()
+                ):
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)